misc/alert_rules.json

```diff
@@ -161,5 +161,25 @@
   {
     "rule": "%sensors.sensor_current < %sensors.sensor_limit_low && %sensors.sensor_alert = \"1\" && %macros.device_up = \"1\" && %macros.sensor_port_link = \"1\"",
     "name": "Sensor under limit with linked port"
+  },
+  {
+    "rule": "%sensors.sensor_current = \"4\" && %sensors.sensor_type = \"upsOutputSourceState\"",
+    "name": "UPS (RFC1628 compatible) is running on the bypass"
+  },
+  {
+    "rule": "%sensors.sensor_current = \"5\" && %sensors.sensor_type = \"upsOutputSourceState\"",
+    "name": "UPS (RFC1628 compatible) is running on the battery"
+  },
+  {
+    "rule": "%sensors.sensor_current = \"3\" && %sensors.sensor_type = \"upsBatteryStatusState\"",
+    "name": "UPS (RFC1628 compatible) has a low battery"
+  },
+  {
+    "rule": "%sensors.sensor_current = \"4\" && %sensors.sensor_type = \"upsBatteryStatusState\"",
+    "name": "UPS (RFC1628 compatible) has a depleted battery"
+  },
+  {
+    "rule": "%sensors.sensor_descr ~ \"Percentage load\" && %sensors.sensor_current >= \"90\"",
```

Comment:
This one will pick up anything that's got that descr which we can't guarantee it will be ups related. Can you not use sensor_type like the others?

Reply:
Changed, but RFC1628 uses a generic type for all.